jax/_src/api.py

```diff
@@ -1595,8 +1595,16 @@ def _get_axis_size(name: str, shape: Tuple[int, ...], axis: int):
     if kws:
       # if keyword arguments are included in the tree, we make adapt the error
```

Comment:
Is this comment relevant anymore?

Reply:
Yes, it applies to the following line where `tree` is rewritten with a copy that does not include keyword arguments.